clock.py

```diff
@@ -0,0 +1,163 @@
+import datetime
+import os
+import time
+
+
+
+dict_num=\
```

Comment:
I would pre-split the digits into raws

[
"xxxx",
"x    x"
]

Reply:
ok, inderstand that